File: app/src/app/core/models/answer.model.ts
export class AnswerModel {
    id: number;
    answerId: string;
    sectionId: string;
    questionId: string;
    answer: string;
    expectedText: boolean;
    jumpToQuestion: string;
    isInvalid: boolean;
    isChecked: boolean;
    answerTxt: string;
}

File: Code/Foundation/Models/AnswerModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Foundation.Models
{
    public class AnswerModel
    {
        public int Id { get; set; }
        public string AnswerId { get; set; }
        public string SectionId { get; set; }
        public string QuestionId { get; set; }
        public string Answer { get; set; }
        public bool ExpectedText { get; set; }
        public string JumpToQuestion { get; set; }
    }
}
